src/test/java/com/google/api/generator/gapic/composer/grpc/ServiceClientTestClassComposerTest.java

```diff
@@ -30,15 +30,16 @@ public class ServiceClientTestClassComposerTest {
   public static Collection<Object[]> data() {
     return Arrays.asList(
         new Object[][] {
-          {"EchoClientTest", GrpcTestProtoLoader.instance().parseShowcaseEcho(), 0},
+          {"EchoClientTest", GrpcTestProtoLoader.instance().parseShowcaseEcho(), 0, ""},
```

Comment:
We don't need to touch this test since we didn't touch `ServiceClientTestClassComposer` or `AbstractServiceClientTestClassComposer`

Reply:
Removed